### DISCUSSION TITLE
end g-code (coming from prusa slicer)

### DISCUSSION THREAD

**Discussion by Smirzor** [supermerill/SuperSlicer#1100] [Q&A] [answered]
i would quite like to switch over from prusaslicer. There i have the following end gcode macro in my printer settings
judging by the error message in the log superslicer does not use the variable max_layer_z and print_bed_max[1]. I was just wondering if there is an equivalent to this?

`{if max_layer_z < max_print_height}G1 Z{z_offset+min(max_layer_z+2, max_print_height)} F600 ; Move print head up{endif}
G1 X5 Y{print_bed_max[1]*0.8} F{travel_speed*60} ; present print
{if max_layer_z < max_print_height-10}G1 Z{z_offset+min(max_layer_z+70, max_print_height-10)} F600 ; Move print head further up{endif}
{if max_layer_z < max_print_height*0.6}G1 Z{max_print_height*0.6} F600 ; Move print head further up{endif}
M140 S0 ; turn off heatbed
M104 S0 ; turn off temperature
M107 ; turn off fan
M84 X Y E ; disable motors`

**Comment by Smirzor** [comment, marked as answer]
I changed max_layer_z with layer_z to solve that issue
for now I removed the print_bed_max[1] as that's not that important

**Comment by supermerill** [comment]
that's something that I may have missed in a merge. You can create an issue to report these kinds of missing things.
